roles/installer/molecule/satellite/converge.yml

```diff
@@ -1,12 +1,13 @@
 ---
 - name: Converge
   hosts: all
-  gather_facts: false
+  gather_facts: true
   vars:
-    foreman_installer_scenario: foreman
+    satellite_installer_scenario: foreman
+    satellite_installer_package: foreman-installer
```

Comment:
this feels like cheating, esp as it now doesn't test that we have a sane default for `satellite_installer_package`?

Reply:
Without a satellite-installer package to actually install, what do you suggest?

Reply:
We could build a fake package, that's the only thing that comes to my mind. Whether it's a good idea or not is a different question ;)

Reply:
Can you add a role to the collections source code that is not included in the collection itself? (for sharing amongst testing since this would get re-used over and over)

Reply:
Yeah.

You can either add [`build_ignore`](https://docs.ansible.com/ansible/latest/dev_guide/collections_galaxy_meta.html) to the `galaxy.yml`, but that requires ansible 2.10+ for building the collection (which we have when we release upstream using GH Actions, but you'd need to remember to use when executing it on your Fedora for Satellite).

Or you hack up the `Makefile` not to copy that one role when it does `dist`.